src/App.tsx

```diff
@@ -1,22 +1,43 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
 export const App: React.FC = () => {
+  const [query, setQuery] = useState('');
+
+  const getMovies = (movies: Movie[], userQuery: string) => {
+    const lowerCaseQuery = userQuery.toLowerCase().trim();
+
+    return movies.filter(movie => {
+      const lowerCaseTitle = movie.title.toLowerCase();
+      const lowerCaseDescription = movie.description.toLowerCase();
+
+      return lowerCaseTitle.includes(lowerCaseQuery)
+        || lowerCaseDescription.includes(lowerCaseQuery);
+    });
```

Comment:
like idea with `getMovies`, looks pretty, but still duplicated
DRY, create a function to check if some movie property matches query